config/locales/en.yml

```diff
@@ -1,4 +1,32 @@
 en:
+  features:
+    about:
+      short: About
+      long: About
+    events:
+      short: Events
+      long: Events
+    materials:
+      short: Materials
+      long: Training materials
+    elearning_materials:
+      short: e-Learning
+      long: e-Learning
+    workflows:
+      short: Workflows
+      long: Workflows
+    collections:
+      short: Collections
+      long: Collections
+    trainers:
+      short: Trainers
+      long: Trainers
+    content_providers:
+      short: Providers
+      long: Content Providers
+    nodes:
+      short: Nodes
+      long: Nodes
```

Comment:
If you're planning on renaming features, you may want to use the Rails approach: 
https://github.com/ElixirTeSS/TeSS/blob/01e4227513abc677532078891c5a3743847cf0bb/config/locales/en.yml#L37-L39

This will make sure it gets used in things like validation error messages etc.